### PR TITLE
Additional game help

### PR DESCRIPTION
**Pull request by badvision**
Trying to make a little dent in #637 

Covers:
- Aliens
- Aquatron
- Blitzkreig
- Bubble Head
- California Raisins
- Copts and Robbers (clarification) 
- Diamond Mine
- Dragonfire
- Gauntlet
- G.I. Joe
- Matterhorn Screamer
- Serpentine

### CODE REVIEW

**Comment by badvision** [comment]
Ok after some cleanup and corrections I'm pretty much set with my change set now.  Make sure to squash/merge this since the commit history is not likely very useful or interesting to anyone.

**Comment by badvision** [comment]
I've reduced the content to fit the double space requirement, trying to sacrifice as little meaning as possible.  Let me know if this works.

File: res/GAMEHELP/ALIENS [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+ Ops room: Keep aliens away long enough
+
+
+ Maze: Get out, save as many as you can
+
+
+             End of level:
+
+   Return * * * * * * * * *Next Level
```
Comment (a2-4am) [inline res/GAMEHELP/ALIENS:21]: These need to span the entire width (40 columns)

File: res/GAMEHELP/AQUATRON [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+  of the screen will engage hyperdrive
+
+ Activate joystick with B0 during play
+
+                   I
+           move  J   L   move
+                   K
+
+f / B0 * * * * * machine guns (unlimited)
```
Comment (a2-4am) [inline res/GAMEHELP/AQUATRON:19]: capital F, capital D, capital E in Esc

File: res/GAMEHELP/BLITZKRIEG [hunk 3]
```diff
@@ -0,0 +1,19 @@
+
+
+
+
+     "SHOOT EM DOWN WITH YOUR GUN."
```
Comment (a2-4am) [inline res/GAMEHELP/BLITZKRIEG:5]: no all-caps

File: res/GAMEHELP/BLITZKRIEG [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+
+  Using paddles or keyboard shoot down
+
+enemy planes and any bombs they may drop
+
+       until you run out of ammo.
+
+
+u or joystick right* * * * * * * * aim up
```
Comment (a2-4am) [inline res/GAMEHELP/BLITZKRIEG:15]: capital U, capital D, capital F

File: res/GAMEHELP/BUBBLE.HEAD [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+ Extra life is awarded at 10,000 points
+
+   Joyport for movement, or keyboard:
+
+             I            A
+           J   K   or   <   >
+             M            Z
+
+  space / B0 * * * * * * * Force Field
```
Comment (a2-4am) [inline res/GAMEHELP/BUBBLE.HEAD:20]: expand to full 40-columns

File: res/GAMEHELP/BUBBLE.HEAD [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+  ctrl-r * * * * * * * * Restart Level
```
Comment (a2-4am) [inline res/GAMEHELP/BUBBLE.HEAD:22]: `Ctrl-R`

`restart level`

File: res/GAMEHELP/BUBBLE.HEAD [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+  esc* * * *Pause    ctrl-s* * * Sound
```
Comment (a2-4am) [inline res/GAMEHELP/BUBBLE.HEAD:24]: `Esc`

`Ctrl-S`

File: res/GAMEHELP/CALI.RAISINS [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+
+        bridge for extra points
+
+                   I
+          move   J   K   move
+                   M
+
+Return/B0 * *Jump    Space/B1 * * *Fire
+
+CTRL+J* *Joystick   ESC * * *Next Level
```
Comment (a2-4am) [inline res/GAMEHELP/CALI.RAISINS:20]: `Ctrl-J`

`Esc`

File: res/GAMEHELP/CR [hunk 9]
```diff
@@ -13,11 +14,8 @@
     Space to pick up or drop object
 
 
+esc * * * * pause   0-7 * * * game speed
 
-Esc* * * * * * * * * * * * * * * * pause
+ctrl-c* * *config   ctrl-r* * * *restart
```
Comment (a2-4am) [inline res/GAMEHELP/CR:19]: `Ctrl-C`

`Ctrl-R`

File: res/GAMEHELP/CR [hunk 10]
```diff
@@ -13,11 +14,8 @@
...
 
-Ctrl-C * * * * * * * * * * * change keys
-
-Ctrl-R * * * * * * * * * * * * * restart
-
-0-7* * * * * * * * * * * * * *game speed
+z * * * * suicide   R * * *reincarnation
```
Comment (a2-4am) [inline res/GAMEHELP/CR:21]: `Z`

File: res/GAMEHELP/DIAMOND.MINE [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+
+        to steal them from you!
+
+    P1   I                   W   P2
+       J   K               A   D
+  Move   L                   S   Move
+
+ Shoot  Space                1   Shoot
+
+ctrl-b * * * * restart with new options
```
Comment (a2-4am) [inline res/GAMEHELP/DIAMOND.MINE:15]: `Ctrl-B` and similar capitalization

File: res/GAMEHELP/DRAGONFIRE [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+
+         move   J     L   move
+
+                N  M  ,
+
+        K to duck high fireballs
+
+  Space / Button to jump low fireballs
+
+0-9 * * difficulty  esc * * * * * pause
```
Comment (a2-4am) [inline res/GAMEHELP/DRAGONFIRE:21]: capitalization changes like others

File: res/GAMEHELP/G.I.JOE [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+
+                   I
+           move  J   K  move
+                   M
+
+Enter * * * * * * start selected mission
+
+Button 0 or space * * * * * * * * *shoot
+
+Button 1 or ctrl + direction* * * * *aim
```
Comment (a2-4am) [inline res/GAMEHELP/G.I.JOE:19]: `Ctrl`

File: res/GAMEHELP/GAUNTLET [hunk 14]
```diff
@@ -0,0 +1,22 @@
...
+
+move    J   L             D   S     move
+
+        N M .             Z X C  
+
+   B1/$ or K     shoot       B0/% or D
+
+     space       magic          tab
+
+      ctrl-p * * * * * * * * pause
```
Comment (a2-4am) [inline res/GAMEHELP/GAUNTLET:22]: capitalization and width

File: res/GAMEHELP/MS [hunk 15]
```diff
@@ -0,0 +1,23 @@
...
+    J   K   or   <   >   or   <   >
+      M            Z            
+
+  Up/Down jump up and down ledges, but
+
+ the jump button is for jumping enemies
+
+ Space or Button 0 * * * * * * * * Jump
+
+ ESC * * * * * * * * * * * * * * *Pause
```
Comment (a2-4am) [inline res/GAMEHELP/MS:23]: `Esc`

`pause`

File: res/GAMEHELP/MS [hunk 16]
```diff
@@ -0,0 +1,23 @@
...
+      I            A            

+    J   K   or   <   >   or   <   >
+      M            Z            
+
+  Up/Down jump up and down ledges, but
+
+ the jump button is for jumping enemies
+
+ Space or Button 0 * * * * * * * * Jump
```
Comment (a2-4am) [inline res/GAMEHELP/MS:21]: `jump`

File: res/GAMEHELP/SERPENTINE [hunk 17]
```diff
@@ -0,0 +1,24 @@
...
+
+       Eat frogs to grow longer
+
+        and turn enemies green.
+
+                   A
+          move   <   >   move
+                   Z
+
+c* * * * * * * * * * change key mappings
```
Comment (a2-4am) [inline res/GAMEHELP/SERPENTINE:18]: `C`

File: res/GAMEHELP/SERPENTINE [hunk 18]
```diff
@@ -0,0 +1,24 @@
...
+
+return * * * * * * view high score table
```
Comment (a2-4am) [inline res/GAMEHELP/SERPENTINE:20]: `Return`

File: res/GAMEHELP/SERPENTINE [hunk 19]
```diff
@@ -0,0 +1,24 @@
...
+
+esc* * * * * * * * * * * * * * * * pause
```
Comment (a2-4am) [inline res/GAMEHELP/SERPENTINE:22]: `Esc`

**Review by a2-4am — CHANGES_REQUESTED** [review]

**Comment by badvision** [comment]
Thanks for your exceeding patience!  Let me know if these changes work for you.

**Comment by a2-4am** [comment]
Thank you!

**Pull request merged**